### PR TITLE
Added exporting logs script

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: export-logs.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+headers = {"Authorization": f"Token {api_token}"}
+
+# API-endpoint for environments
+ENV_URL = "https://api.divio.com/apps/v3/environments/"
+
+# Customer-provided values for the script
+from_ts = "YYYY-MM-DDThh:mm"  # Replace with desired start date and time
+to_ts = "YYYY-MM-DDThh:mm"  # Replace with desired end date and time
+env_slug = ""  # Replace with the environment slug (e.g., "live" or "test")
+app_uuid = ""  # Replace with the application UUID
```
Comment (atifazad) [inline export-logs.py]: It's better to accept these value as command line arguments so that the user doesn't have to open and modify the file every time.

Comment (mebverse) [reply export-logs.py]: I have updated the script, let me know if that is okay to update the Docs accordingly.

**Comment by atifazad** [comment]
@mebdivio your commits are not verified.

File: export-logs.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+    # Open a file to save the logs (replace 'logs.txt' with your desired file name)
+    with open("logs.txt", "w") as log_file:
+
+        # Main loop to retrieve and display logs
+        while True:
+            # Fetch the log data from the current URL
+            data = response.json()
+
+            # Check for any error messages in the response
```
Comment (atifazad) [inline export-logs.py:136]: please correct the comment

File: export-logs.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    Returns:
+        str: The truncated timestamp without microseconds (up to milliseconds precision).
+    """
+    if "." in timestamp:
+        return timestamp[: timestamp.index(".") + 3]  # Truncate to two decimal places
+    return timestamp
+
+
+# Function to export and display logs of a given environment
+def get_all_logs(env_uuid, headers):
```
Comment (atifazad) [inline export-logs.py]: please rename to `get_logs`

File: export-logs.py (hunk 2 above)
Comment (mebverse) [reply export-logs.py:136]: done, it is correct.

File: export-logs.py (hunk 3 above)
Comment (mebverse) [reply export-logs.py]: done it is renamed

**Pull request merged**